docs/en/admins/08_FeedUpdates.md

```diff
@@ -6,6 +6,7 @@ FreshRSS is updated by the `./app/actualize_script.php` script. Knowing this, we
 
 **Note:** the following examples assume that FreshRSS is installed to `/usr/share/FreshRSS`. You'll need to modify the FreshRSS path to reflect your own system.
 
+**Note:** If you can't configure a Crojob look here for an alternative https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron
```

Comment:
```suggestion
**Note:** If you cannot configure a Cronjob, [look here for an alternative](../users/03_Main_view.md#online-cron).
```

Reply:
I'd make that "an alternative using online cron" or something along those lines.